website/src/docs/webcam.md

```diff
@@ -64,15 +64,15 @@ uppy.use(Webcam, {
     'picture',
   ],
   mirror: true,
+  showVideoSourceDropdown: false,
+  facingMode: 'user', // @TODO: remove in the next major
```

Comment:
I think it would be more useful to show the deprecation notice here, the TODO is only for us, so it should not appear in the docs.